skbio/diversity/_fast_base_cy.pyx

```diff
@@ -0,0 +1,215 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import numpy as np
+cimport numpy as np
+cimport cython
```

Comment:
Typically we've been `cimport(ing) numpy as cnp` because it makes it easier for newer Cython developers to understand what happens at translation/compile time vs what happens at runtime, although it's absolutely not necessary to use a different alias.